src/app/core/core.module.ts

```diff
@@ -0,0 +1,19 @@
+import { NgModule, ModuleWithProviders } from '@angular/core';
+import { CommonModule } from '@angular/common';
+import { MAT_SNACK_BAR_DEFAULT_OPTIONS } from '@angular/material/snack-bar';
+
+@NgModule({
+  declarations: [],
+  imports: [CommonModule],
+  providers: [
+    {
+      provide: MAT_SNACK_BAR_DEFAULT_OPTIONS,
```

Comment:
Maybe it would be better to move this to the `material` module?

Reply:
Okaaaay